example/l2fwd/README

```diff
@@ -5,3 +5,10 @@ Source code and Makefiles placed under test/performance/ directory.
 This L2 forwarding application can be used as example reference as well
 as performance test for different odp modes (direct, queue or scheduler
 with parallel, atomic or ordered queues).
+
+Note that this example is tuned for performance. As a result, when using
+scheduled mode with direct packet I/O output or queued output with multiple
+output queues, packet order is not guaranteed. To achieve guaranteed order,
+use a single worker thread or output interfaces with single output
+queues. Other examples of scalable processing using ordered queues that
+preserve order can be seen in the odp_pktio_ordered performance test.git
```

Comment:
Looks ok to me now.

It seem there is accidental text "git" at the very end.

Reply:
Hmmm. Not sure how that stray got in there. Fixed in v4.